docs/en/Get-Started.md

```diff
@@ -78,13 +84,13 @@ $ echo "alluxio.master.hostname=localhost" >> conf/alluxio-site.properties
 
 Set additional parameters in `conf/alluxio-site.properties`
 ```console
-echo "alluxio.dora.client.read.location.policy.enabled=true" >> conf/alluxio-site.properties
-echo "alluxio.user.short.circuit.enabled=false" >> conf/alluxio-site.properties
-echo "alluxio.master.worker.register.lease.enabled=false" >> conf/alluxio-site.properties
-echo "alluxio.worker.block.store.type=PAGE" >> conf/alluxio-site.properties
-echo "alluxio.worker.page.store.type=LOCAL" >> conf/alluxio-site.properties
-echo "alluxio.worker.page.store.sizes=1GB" >> conf/alluxio-site.properties
-echo "alluxio.worker.page.store.page.size=1MB" >> conf/alluxio-site.properties
+$ echo "alluxio.dora.client.read.location.policy.enabled=true" >> conf/alluxio-site.properties
```

Comment:
if we want to enforce that all lines in console code blocks should start with a `$`, there should be a check to help enforce it. cc @Zijian-Zhu